### PR TITLE
fix & refactor: several typos & mis-uses

### PR DESCRIPTION
**Pull request by lobshunter**
A bunch of updates according manual debugging result.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,34 @@
...
 - PD: Resign leader if the current PD instance is the PD leader
 
  An additional container is added to run the maintenance watching script.
 
 ## Deploy
+
 ### Sidecar Image
-Used Sidecar Public image: lobshunter/gcp-live-migration-tikv 
+
+Used Sidecar Public image: lobshunter/gcp-live-migration-tikv
```
Comment (dbsid) [inline README.md]: should we change the image to `tidb-google-maintenance` or `gcp-live-migration`

File: README.md [hunk 2]
```diff
@@ -135,10 +167,17 @@ For PD, add content below to spec.pd (replace ${CLUSTR_NAME}),
...
+
+```SQL
 set config pd `leader-schedule-limit`=100;
 ```
 
 ## Limitation
-Current version is only for cluster deployed by TiDB Operator and TLS is eanbled.
\ No newline at end of file
+
+Current version is only for cluster deployed by TiDB Operator and TLS is eanbled.
```
Comment (dbsid) [inline README.md]: `and TLS is eanbled` should be removed

Comment (lobshunter) [reply README.md]: addressed in [f31451a](https://github.com/PingCAP-QE/tidb-google-maintenance/pull/4/commits/f31451a5c678639b5e2675a484dfa26cf3832de7)

File: README.md (hunk 1 above)
Comment (lobshunter) [reply README.md]: addressed in [f31451a](https://github.com/PingCAP-QE/tidb-google-maintenance/pull/4/commits/f31451a5c678639b5e2675a484dfa26cf3832de7)

**Review by dbsid — APPROVED** [review]

**Pull request merged**